Add Braunschweig GDCR 2018 event

stevenschwenke opened the pull request.

rradczewski commented:
Hi Steven, thanks for adding your event! By looking at the url, I wasn't quite sure how to sign up for the coderetreat. Is there a dedicated page you could send people to?

rradczewski commented on _data/events_gdcr2018/Braunschweig.json:
```diff
@@ -0,0 +1,17 @@
+{
+  "title": "HackCamp 6 Braunschweig",
+  "url": "https://hacktalk.de",
```
Could you please update this PR with the sign up page as soon as it's live?

stevenschwenke commented:
I just added information on how to signup for the event. There will be a Xing event, but right now it's just via mail. Also, I updated the link with an anchor to get directly to the right position on the page.
Happy to get feedback :) Thanks!
